io/plugins/eu.esdihumboldt.hale.io.gml.test/src/eu/esdihumboldt/hale/io/gml/reader/internal/GmlInstanceCollectionTest.java

```diff
@@ -162,8 +163,6 @@ public void testLoadShiporder() throws Exception {
 
 			Object[] orderid = instance.getProperty(new QName("orderid")); // attribute
 																			// form
-																			// not
```

Comment:
I think the original comment referred to that no namespace needs to be provided because the attribute form as specified in the XML schema is not qualified (i.e. `unqualified`). If you remove the comment I would say it makes sense to remove it completely.